components/docs/DocsCards.tsx

```diff
@@ -0,0 +1,61 @@
+import Link from 'next/link';
+import React from 'react';
+
+import { HeadingLevel, HeadingTypeStyle } from '@/types/typography/Heading';
+import { ParagraphTypeStyle } from '@/types/typography/Paragraph';
+
+import { buckets } from '../data/buckets';
+import Heading from '../typography/Heading';
+import Paragraph from '../typography/Paragraph';
+
+type IconType = (props: { className: string }) => JSX.Element;
+
+interface CardProps {
+  title: string;
+  description: string;
+  link: string;
+  className: string;
+  Icon: IconType;
+}
+
+const Card: React.FC<CardProps> = ({ title, description, link, className, Icon }) => {
+  return (
+    <Link href={link}>
+      <a href={link} className='cursor-pointer' data-testid='Docs-link'>
+        <div className='h-full rounded-lg border border-gray-200 p-6 shadow-md transition-all duration-300 ease-in-out
+          hover:shadow-lg'>
```

Comment:
Please add the `legacyBehavior` prop to `Link` here. Eg. `<Link legacyBehavior ...otherProps>` or make a workaround to not nest an Anchor Tag inside <Link>.

Ref: https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor